tests/metrics/functional/test_classification.py

```diff
@@ -120,55 +87,6 @@ def test_get_num_classes(pred, target, num_classes, expected_num_classes):
     assert get_num_classes(pred, target, num_classes) == expected_num_classes
 
 
-@pytest.mark.parametrize(['pred', 'target', 'expected_tp', 'expected_fp',
```

Comment:
Why did we remove those tests ?

Reply:
I am removing tests for deprecated functions. This was also done in other PRs before, see #4704